### PR TITLE
[FEAT]: Add remove and open button in reminder's modal 

### PR DESCRIPTION
**Pull request by VipinDevelops**
## Acceptance Criteria fulfillment

- [x] Added "Remove" and "Open" buttons in the reminder modal for each repositry.

## Proposed changes (including videos or screenshots)
[Screencast from 2023-10-25 15-55-43.webm](https://github.com/RocketChat/Apps.Github22/assets/99081689/558ecfef-608f-40d1-85db-48a74b18a423)

### CODE REVIEW

File: github/persistance/remind.ts [hunk 1]
```diff
@@ -117,3 +117,14 @@ export async function getUserReminder(read:IRead,User:IUser):Promise<IReminder>{
     const index = reminders.findIndex((reminder)=>reminder.userid === User.id);
     return reminders[index];
 }
+
+export async function removeRepoReminder(read:IRead,persistence:IPersistence,repository:string,User:IUser){
+    const reminders = await getAllReminders(read);
+
+    const idx = reminders.findIndex((u: IReminder) => u.userid === User.id);
+    const repoindex = reminders[idx].repos.findIndex((repo)=>repo == repository);
```
Comment (samad-yar-khan) [inline github/persistance/remind.ts:125]: @VipinDevelops what if findIndex finds now value and we get invalid index ?

**Review by samad-yar-khan — CHANGES_REQUESTED** [review]

File: github/persistance/remind.ts [hunk 2]
```diff
@@ -117,3 +117,16 @@ export async function getUserReminder(read:IRead,User:IUser):Promise<IReminder>{
...
 }
+
+export async function removeRepoReminder(read:IRead,persistence:IPersistence,repository:string,User:IUser){
+    const reminders = await getAllReminders(read);
+    const idx = reminders.findIndex((u: IReminder) => u.userid === User.id);
+    const repoindex = reminders[idx].repos.findIndex((repo)=>repo == repository);
+
+    if (idx === -1 || repoindex === -1) {
+        return;
+    }
```
Comment (samad-yar-khan) [inline github/persistance/remind.ts:128]: if idx is -1 this will still break at line 124

**Review by samad-yar-khan — CHANGES_REQUESTED** [review]

File: github/persistance/remind.ts (hunk 2 above)
Comment (VipinDevelops) [reply github/persistance/remind.ts]: I fixed it @samad-yar-khan

**Review by samad-yar-khan — APPROVED** [review]
LGTM ! thanks @VipinDevelops

**Pull request merged**